src/app/clusters/bindings/BindingManager.cpp

```diff
@@ -18,29 +18,68 @@
 #include <app/clusters/bindings/BindingManager.h>
 #include <app/util/binding-table.h>
 
+namespace {
+
+chip::PeerId PeerIdForNode(chip::FabricTable & fabricTable, chip::FabricIndex fabric, chip::NodeId node)
+{
+    chip::FabricInfo * fabricInfo = fabricTable.FindFabricWithIndex(fabric);
+    if (fabricInfo == nullptr)
+    {
+        return chip::PeerId();
+    }
+    return fabricInfo->GetPeerIdForNode(node);
+}
+
+} // namespace
+
 namespace chip {
 
 BindingManager BindingManager::sBindingManager;
 
+CHIP_ERROR BindingManager::UnicastBindingCreated(uint8_t bindingEntryId)
```

Comment:
Is there a reason to not make this take a cost ref to the binding table entry?  The caller has one (on stack)....  And we never use the id here except to get the entry.